src/status_im/ui/screens/bottom_sheets/views.cljs

```diff
@@ -28,6 +28,9 @@
           (= view :add-new)
           (merge home.sheet/add-new)
 
+          (= view :add-new2)
```

Comment:
I often find myself writing "Add new..." but wait, adding something is already a new thing ;) Nothing to change here, but whenever possible I usually stick with just "Add" or if add is too generic, "add \<thing\>".

Reply:
Changed to `add-new-sheet-view`